cmd/gateway-router.go

```diff
@@ -44,113 +38,4 @@ type Gateway interface {
 // GatewayLayer - interface to implement gateway mode.
 type GatewayLayer interface {
```

Comment:
` type GatewayLayer interface` needs to be removed

Reply:
done